src/liveShare/shareSession.ts

```diff
@@ -43,7 +46,17 @@ export interface IRequest {
         command: string,
         start_time: string
     };
-}
+} & (
+    {
+        coomad: 'dataview';
```

Comment:
Seems like a typo?

Reply:
It was my attempt to use [typescript type discramination](https://www.typescriptlang.org/play?#code/PTAEEFQEwSwZwMYCcYFsYDsCGAXAplKDgJ4AOeoArhjAPYajygDuAFnkhcbZVXBQlpQKAMwA2tZgCgQoLNjHE4THLVCcolBBSyh+OULRGhStfBhwwsYwwCMAVngQ440SQ1WGMeGWDjkEGBEYBDtHZwA6X19QABVWJlJcfCQGZlokAGtXTmtFFjwxGxEMvTwXQ2NldDEsJBjaBycK5hgcVjloIJEOPAs9HBQMAHNDJFAMSlRbDlBBDDgceRwYkvG8AA8sVFIxPAAuTrF4AyMJ7YJQPAA3PpcAGjHQW6RlegIY-VczxvCXKKkJHIcTQmGGAFFbv0ALygADe51QB1AACJFnUcCiANx8DgAZSWSHwUEOtlotD28lAAF9QAAfeGI5EohASfhQbHQShIXB0DCHSbTWbUrFSGIAdXYDBudzmtCRrkwnnaTBE1GcfMebTllDEhBmcgwxAaxnaFBw6RMZjuVhsQLwcAB80WoFY8ige0hsthAAoZRZDrFQSMvRYAJSgaEAPnhUlAoFkACFiHwwZ04K0cAgOlhhlhMC7-TgItgkXEyHg8cgYKQcAByVyCYRxhNgcSSQ3WJRMBBU4QpdDeIhugxU37NOZjjCKFuyA2cUicfgWS62FPvIgVxgedhUGj0AHxjNtbOgP1Q4ulvARuEt+O9-io9FElH7O-x1txBKuJHyVzcXhewYOAsB6fIsAQbQ4FcSh+CQAkMQ+D8PznJwsFgig2gbYcKHofJ7W3ZRhBBQcIQvd941kNheiZRhXDRQlMQouUFgMTA2isYlkwAVTgyMrgvCIMPgxiCFFZD41sXJMlFd8HwoFk2QIV9mOdAxLAVJIGFhIsIk0HlLHocSJKkvAsBkltqSkEUxVkeJEmSDgGBUXcQLLTMOkFGZXhYBJT2YAQqQwmIsFcM0ukQFBB1wDIAVkABJHcmE2bZdjwR4AtdLBbk6WBIuDXAKGoPlDUIeQQqgWBDIUK4kCQUp0XwIg1DdDAPTwAECPAAAFeKACUHVMBYHX4hEXjeflQAABhxVA4GGQ5FiGUZaQZMaOAmw4AEZZodEDhmRJawRxRrYIFKZvJpelGQ4eqkEWwY0xsqQ1Kytq9gG-x6EfX0lyG-hDh6-rBu+h0I2jWNKLAAAJd09hw2q7snfhHndHCGE4HBuQwFsgjPFFboyFFt3UEHhpvOTvopDrCaQH0-tBiJabDYz40x7HjKs2cwFiCsqxQWsJg7TIMEkMKRwgXrPv+wKMFFgwZm5hHac3cgIlAeLjG1ALOGVuqMkecK1QwDV6CV9JdUIN0cvZ1ICHVgBNHhJwYF4ghTFVXDXJXWFoN2RkMF5Sa+4bQBmCRmEPRhjHpsn+AicaSuhZPpop5DnWpiIJGGWOQ-jubhhZyyrjER88dzmWE42pOU62tOPwzvYs9oHOGeGiJTrgR42-zvbc2vTnrNk2QncoOtdZmHAUkqYxYLTXRjyzDpGrwJF+gLfAsEIIwYjxle14qGZeww0AAJd0BUHMnRoO5eQoOHXA5CKGIkiJb5TV3Yr6DkXXsycTJ7ZfhokwXQwxySECSFPJyqxSie1TAHcK3gg4EUwAjVqUAyS0EyG+GIrAp6kDgPsEAzASERHtPlWstQRgRAyMMYAUBaAIDgMAdBmDMjAC3tcO+BAAC05CIh4NQGIAAxHlas0ViQ8K-gsKQQA).
However, I have many type design issues for now, so I'm rolling back from this attempt. You can use this typing strategy later if you want to.